### PR TITLE
Styles/h teacher bug

### PR DESCRIPTION
**Pull request by helenemccarron**
Pivotal ticket https://www.pivotaltracker.com/n/projects/1156756/stories/100824216

### CODE REVIEW

File: resources/styles/book-content/os-teacher.less [hunk 1]
```diff
@@ -1,6 +1,12 @@
 .os-teacher {
-  &:before {
-    color: @tutor-neutral-dark;
-    background: @tutor-quaternary;
-  }
-}
\ No newline at end of file
+	&:before {
```
Comment (philschatz) [inline resources/styles/book-content/os-teacher.less]: Are those tabs? ; )

**Comment by helenemccarron** [comment]
what tab ;P

**Comment by helenemccarron** [comment]
actually, after fixing the display-inline for all the os-notes,  I think tis isn't necessary anymore >.<;

**Comment by philschatz** [comment]
If this PR is a duplicate can you close it?

**Pull request closed without merging**